.github/workflows/link_validation.yaml

```diff
@@ -35,6 +35,6 @@ jobs:
           python3 -m pip install --upgrade pip
           pip3 install setuptools wheel twine tox mechanical-markdown
       - name: Check Markdown Files
+        continue-on-error: true
         run: |
-          for name in `find . -name "*.md"`; do echo -e "------\n$name" ; mm.py -l $name || exit 1 ;done
-
+          for name in `find . -name "*.md"`; do echo -e "------\n$name" ; mm.py -l $name ;done
```

Comment:
Applied the same solution that I did to the v1.7 docs, given that v1.6 are also legacy docs. Link-checking will report errors but not fail the build.